packages/cli/src/commands/hydrogen/deploy.ts

```diff
@@ -291,6 +377,11 @@ export async function oxygenDeploy(
           ],
         ],
       });
+      // in CI environments, output to a file so consequent steps can access the URL
+      // the formatting of this file is likely to change in future versions.
+      if (isCI && !noJsonOutput) {
+        await writeFile('h2_deploy_output.log', JSON.stringify({url: url!}));
```

Comment:
Perhaps the log should end in `.json`? E.g. `h2_deploy_log.json` or `h2_deploy.log.json`